package.json

```diff
@@ -42,5 +42,8 @@
   "engines": {
     "node": ">=16.0.0"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "dependencies": {
+    "sketchpad": "^0.1.0"
```

Comment:
This dependency should be added to apps/prairielearn, not the root.

Reply:
agreed, removed and added there.